pod-operation/src/utils/mod.rs

```diff
@@ -0,0 +1,2 @@
+mod gpio;
+pub use gpio::GpioPins;
```

Comment:
Is it not enough to just have `mod gpio` like we have in the `mod.rs` of `components`? What is this line used for?